easybuild/easyconfigs/a/Amber/Amber-20.11-AmberTools-20.15-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,54 @@
+name = 'Amber'
+local_amber_ver = 20
+local_ambertools_ver = 20
+# Patch levels from http://ambermd.org/AmberPatches.php and http://ambermd.org/ATPatches.php
+patchlevels = (15, 11)  # (AmberTools, Amber)
+version = '%s.%s-AmberTools-%s.%s' % (local_amber_ver, patchlevels[1], local_ambertools_ver, patchlevels[0])
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ambermd.org/amber.html'
+description = """Amber (originally Assisted Model Building with Energy Refinement) is software for performing
+ molecular dynamics and structure prediction."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+# Amber CMake fails when 'usempi': True is set and we add '-DMPI=TRUE'
```

Comment:
Then we might need to wrap up a patch to fix that, and send it upstream...

Reply:
The error I am seeing is
```
-- Found MPI_C: /rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/OpenMPI/4.0.3-GCC-9.3.0/bin/mpicc (found version "3.1") 
-- Found MPI_CXX: /rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/OpenMPI/4.0.3-GCC-9.3.0/bin/mpicxx (found version "3.1") 
-- Found MPI_Fortran: /rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/OpenMPI/4.0.3-GCC-9.3.0/bin/mpifort (found version "3.1") 
-- Found MPI: TRUE (found version "3.1")  
-- MPI C Compiler: /rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/OpenMPI/4.0.3-GCC-9.3.0/bin/mpicc
-- MPI CXX Compiler: /rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/OpenMPI/4.0.3-GCC-9.3.0/bin/mpicxx
-- MPI Fortran Compiler: /rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/OpenMPI/4.0.3-GCC-9.3.0/bin/mpifort
-- If these are not the correct MPI wrappers, then set MPI_<language>_COMPILER to the correct wrapper and reconfigure.
CMake Error at cmake/LibraryTracking.cmake:174 (message):
  Incorrect usage.  At least one LIBRARY should be provided.
Call Stack (most recent call first):
  cmake/MPIConfig.cmake:122 (import_libraries)
  CMakeLists.txt:118 (include)
```

Reply:
This https://gitlab.kitware.com/cmake/cmake/-/issues/19427#note_589242 looks like issue - that `MPI_C_LIBRARIES` (and CXX and Fortan) are not set, but Amber expects them to be.

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/2